examples/src/main/java/io/numaproj/numaflow/examples/sideinput/simple/SimpleSideInput.java

```diff
@@ -0,0 +1,38 @@
+package io.numaproj.numaflow.examples.sideinput.simple;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.numaproj.numaflow.sideinput.Message;
+import io.numaproj.numaflow.sideinput.Server;
+import io.numaproj.numaflow.sideinput.SideInputRetriever;
+
+public class SimpleSideInput extends SideInputRetriever {
+    private final Config config;
+    private final ObjectMapper jsonMapper = new ObjectMapper();
+
+    public SimpleSideInput(Config config) {
+        this.config = config;
+    }
+
+    @Override
+    public Message retrieveSideInput() {
+        byte[] val;
+        if (0.9 > config.getSampling()) {
```

Comment:
Please add a README for simple side input introducing how it generates side input values. 0.9F, 0.5F and 0.01F are magic numbers.